addons/account/models/account_payment.py

```diff
@@ -419,6 +415,20 @@ def _compute_partner_id(self):
             else:
                 pay.partner_id = pay.partner_id
 
+    @api.depends('journal_id', 'payment_type')
+    def _compute_outstanding_account_id(self):
+        for pay in self:
+            if pay.payment_type == 'inbound':
+                pay.outstanding_account_id = (pay.payment_method_id.payment_account_id
+                                              or pay.journal_id.company_id.account_journal_payment_debit_account_id
+                                              or pay.journal_id.default_account_id)
```

Comment:
I don't know if we should keep this fallback on `default_account_id`. This could be very dangerous for the end user regarding the bank statements management